eml/services/utilityFunctions.js

```diff
@@ -252,22 +251,27 @@ export function findIndexOfUncompletedComp(student, courseId, sectionId) {
 }
 
 export async function handleLastComponent(comp, course, navigation) {
-  const student = await StorageService.getStudentInfo();
-  const isComplete = isSectionCompleted(student, comp.parentSection);
+
+  // For future reference
+  // const student = await StorageService.getStudentInfo();
+  // const isComplete = isSectionCompleted(student, comp.parentSection);
```

Comment:
out of curiosity, what does this mean? :))

Reply:
the future reference stuff is left there for next year's students who will get a PBI for retrying exercises.